HerPortal.BusinessLogic/Models/LocalAuthority.cs

```diff
@@ -1,10 +0,0 @@
-﻿namespace HerPortal.BusinessLogic.Models;
```

Comment:
Hmm, I'm not convinced about dropping this. It will work for now, but there are a couple of things that might come up in the future.

1. I don't think array columns work in SQL server (and may not in other database engines)
2. It makes finding the users associated with an LA difficult in a way that will surprise people that are used to normalised data